src/decisionengine_modules/Rigetti/sources/Rigetti_Balance.py

```diff
@@ -0,0 +1,101 @@
+# SPDX-FileCopyrightText: 2022 Fermi Research Alliance, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Get allocation info from Rigetti
+"""
+from pathlib import Path
+
+import httpx  # just for the typing information
+import pandas as pd
+import qcs_api_client.models as qcs_models
+import qcs_api_client.operations.sync as qcs_sync_operations
+
+from qcs_api_client.client import build_sync_client, QCSClientConfiguration
+
+from decisionengine.framework.modules import Source
+from decisionengine.framework.modules.Source import Parameter
+
+QCS_BASE_PATH = Path("~/.qcs").expanduser()
+
+DEFAULT_SECRETS_FILE_PATH = QCS_BASE_PATH / "secrets.toml"
+DEFAULT_SETTINGS_FILE_PATH = QCS_BASE_PATH / "settings.toml"
+
+__all__ = [
+    "RigettiBalance",
+]
+
+
+@Source.supports_config(
+    Parameter(
+        "constraints",
+        type=dict,
+        comment="""Supports the layout:
+  {
+     'qcs_profile_name': 'default',
+     'qcs_secrets_file_path': '/home/decisionengine/.qcs/secrets.toml',
+     'qcs_settings_file_path': '/home/decisionengine/.qcs/settings.toml',
```

Comment:
Just a comment for the user, anyway. Above you use `Path("~/.qcs").expanduser()`, here `/home/decisionengine/.qcs/`. This may cause inconsistencies if the home dir is different. Why not use the constants defined above?

Reply:
I thought this section was just for documentation of what folks could put in the `jsonnet` config... I'm not sure how to get jsonnet to expand the vars...

Reply:
OK